index.html

```diff
@@ -84,13 +83,17 @@ <h1>Known screens</h1>
 					</tr>
 				</thead>
 				<tbody>
-					<tr>
-						<td colspan="4">Loading…</td>
+					<tr property="screen" mv-multiple mv-if="!search or search(lowercase(name), lowercase(search))>=0 or search(d, search)>=0 or search(w, search)>=0 or search(h, search)>=0 or search(dpi, search)>=0 or search(dppx, search)>=0">
```

Comment:
I think this long expression would be much more readable if it were formatted as multiple lines.